src/language/completionItemProvider.test.ts

```diff
@@ -0,0 +1,44 @@
+import { LinkedToken, SQLMonarchLanguage } from '@grafana/experimental';
+import { getTimestreamCompletionProvider } from './completionItemProvider';
+
+describe('completionItemProvider', () => {
+  describe('when parsing table identifier', () => {
+    const language: SQLMonarchLanguage = {
```

Comment:
why doesn't it make sense to use the real-deal language definition again?

Reply:
In this test, we're actually only testing the `completionItemProvider.parseName` method, and the language is not used there so not relevant for the test. So just passing the minimum required to be a valid language.